python/brayns/client/viewer.py

```diff
@@ -0,0 +1,27 @@
+# Copyright (c) 2015-2022 EPFL/Blue Brain Project
```

Comment:
Are you sure you want to commit that in this PR ?

Because brayns/client has been renamed and I thought we might do it once the high level API is ready to avoid mixing stuff...

Reply:
Removed